doc/api/n-api.md

```diff
@@ -1468,6 +1469,31 @@ This API allocates a `node::Buffer` object and initializes it with data copied
 from the passed-in buffer. While this is still a fully-supported data
 structure, in most cases using a `TypedArray` will suffice.
 
+#### napi_create_date
+<!-- YAML
+added: REPLACEME
+napiVersion: 4
```

Comment:
`napiVersion` shouldn't be set for experimental API's? https://github.com/nodejs/node/pull/28330

Reply:
@richardlau OK, that'll have to be backported as well, then.